src/components/Navbar.jsx

```diff
@@ -0,0 +1,20 @@
+
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
this file shouldn't be included this way, if it must be included add it to index.html file